addons/hr_expense/hr_expense.py

```diff
@@ -112,6 +112,26 @@ def _get_currency(self, cr, uid, context=None):
         'currency_id': _get_currency,
     }
 
+    def add_follower(self, cr, uid, ids,employee_id, context=None):
```

Comment:
- If this method is meant to be used internally, prefix its name with an `_`.
-